Makefile

```diff
@@ -24,6 +24,12 @@ HELP_FUN = \
     }; \
     print "\n"; }
 
+COVPROFILE_UNIT := covprofile-unit
+COVPROFILE_INTEGRATION := covprofile-integration
+COVERED_PACKAGES := ./cmd/...,./internal/...
```

Comment:
Go test coverage doesn't seem to have an option to exclude some directories (like test/fake/ package) so I explicitly specify which packages to be included for coverage.